application/templates/static_site/_preview.html

```diff
@@ -1,5 +1,29 @@
-<div class="cms-preview-banner">
- <span class="message">
-   This is a preview – <a href="https://www.ethnicity-facts-figures.service.gov.uk">Visit live website</a>
- </span>
+<div class="cms-banner">
+  <div class="container-outer">
+    <div class="container">
+      <div class="nav">
+        <div class="left">
+          <span class="application-name">Publisher</span>
+        </div>
+        <div class="middle">
+          <div>
+            <span><a href="https://www.ethnicity-facts-figures.service.gov.uk">Live website</a></span>
+
+            {% if not static_mode and not current_user.is_anonymous and not preview %}
```

Comment:
Won't this entire thing only render outside of static mode? Should that {% if %} be wrapping the entire template? 🤔

Reply:
Or maybe there shouldn't be any reference to static_mode at all in this template, and it should be wrapping the bit of the base template that includes the snippet.

Reply:
@samuelhwilliams you're right, I just copied that condition from the existing template. It think it should be:

```python
if not current_user.is_anonymous and not preview
```

(ie display link to all logged-in users unless they're using a preview link?)